op-challenger/game/fault/contracts/faultdisputegame.go

```diff
@@ -129,6 +129,54 @@ func (f *FaultDisputeGameContract) addGlobalDataTx(ctx context.Context, data *ty
 	return oracle.AddGlobalDataTx(data)
 }
 
+func (f *FaultDisputeGameContract) InitLPP(ctx context.Context, uuid *big.Int, partOffset uint32, claimedSize uint32) (txmgr.TxCandidate, error) {
+	vm, err := f.vm(ctx)
+	if err != nil {
+		return txmgr.TxCandidate{}, err
+	}
+	oracle, err := vm.Oracle(ctx)
+	if err != nil {
+		return txmgr.TxCandidate{}, err
+	}
```

Comment:
I just refactored this into a separate `GetOracle` method so we don't need to duplicate the code over in https://github.com/ethereum-optimism/optimism/pull/9034 Given how much it gets duplicated here we should make sure we wind up using that. 

I think I'd actually not have these methods on `FaultDisputeGame` and just get the oracle from it with that new `GetOracle` and call the oracle contract methods directly.  The large preimage uploader probably shouldn't be tied to the `FaultDisputeGameContract` so makes sense to just pass it the oracle contract.  Having the oracle completely encapsulated made sense when there was only one method but it's adding a lot of boiler plate with all these new methods.

Can land this without it and then follow up though to avoid merge conflicts if you want.

Reply:
Perfect! Let's do that then, I just went ahead since it was a blocker for my follow on work. Will rebase on your changes.